app/Filament/Resources/EventAchievementResource.php

```diff
@@ -157,6 +161,10 @@ public static function form(Form $form): Form
                             ->native(false)
                             ->date(),
 
+                        Forms\Components\TextInput::make('decorator')
+                            ->columnSpan(2)
+                            ->maxLength(96),
```

Comment:
May be worth adding `helperText()` here to explain what this will do once it's added.

Reply:
done